python/tchannel/messages/error.py

```diff
@@ -53,6 +54,6 @@ def error_name(self):
 error_rw = rw.instance(
     ErrorMessage,
     ('code', error_code_rw),                            # code:1
-    ('original_message_id', rw.number(4)),              # id:4
+    ('tracing', common.tracing_rw),                     # tracing:24
```

Comment:
I think tracing_rw only reads 24 bytes for the trace IDs right now. It needs to be modified to read the traceflags and explicit traceflags parsing needs to be removed from everywhere else.

Either that or this needs to read another byte for the traceflags.